app/src/components/pages/collectionPage/collectionPage.tsx

```diff
@@ -68,9 +67,42 @@ const CollectionPage = ({
   const pathname = usePathname();
 
   const updateURL = async (queryString) => {
-    push(`${pathname}?${queryString}`);
-    headingRef.current?.focus();
+    setIsLoaded(false);
+    push(`${pathname}?${queryString}`, { scroll: false });
   };
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
```

Comment:
You have the opportunity to make this a reusable hook if you set a generic `searchManager` argument for the function. Then it can be used here and `searchPage`.